packages/core/src/websockets/reconnecting-websocket.ts

```diff
@@ -416,9 +414,7 @@ export class ReconnectingWebSocket<WS extends IWebSocket = WebSocket>
   }
 
   private _debug(...args: unknown[]): void {
-    if (this._options.debug) {
```

Comment:
Without this guard, the debug logs can get extremely chatty. I don't think we want them on by default. 

I think we should keep the debug option

Reply:
If the issue is that it's not exposed to `MedplumClient` we might want to add a `debug` option to `MedplumClient` too. I know we have an open issue for adding more verbose logging. Maybe the flag could be `verbose` in `MedplumClient` constructor?

https://github.com/medplum/medplum/issues/5009